dd-sdk-android-core/src/test/kotlin/com/datadog/android/core/internal/SdkFeatureTest.kt

```diff
@@ -424,6 +427,33 @@ internal class SdkFeatureTest {
 
     // region FeatureScope
 
+    @Test
+    fun `M unregister datastore W stop()`() {
+        // Given
+        testedFeature.initialize(appContext.mockInstance, fakeInstanceId)
+
+        assertThat(testedFeature.dataStore)
+            .isInstanceOf(DataStoreHandler::class.java)
```

Comment:
this line is useless, because `testedFeature.dataStore` has `DataStoreHandler` type anyway by declaration.